docs/messages_guide/src/asciidoc/_chapters/binder_msgs.adoc

```diff
@@ -2919,3 +2919,15 @@ Use of predefined UDF <name> is deprecated and this function will be removed in
 
 *Recovery:* See message.
 
+[[SQL-4997]]
```

Comment:
Looks like you meant 4497 rather than 4997. But as mentioned above, since this is a run-time error, the error message number should be in the range 8000-8999.